client/src/outside/components/AssignmentsDialog.vue

```diff
@@ -55,13 +77,23 @@ export default {
       selectedSeed: '',
       enableExternalIdEdit: true,
       inUseIds: [],
-      showAdvanced: false
+      showAdvanced: false,
+      persona: '',
+      profession: '',
+      day: 0,
+      time: ''
     }
   },
   components: { AppDialog },
   computed: {
     nameValidate () {
       return this.assignmentName.trim() ? undefined :  ERRORS.NAME_REQUIRED
+    },  
+    timeValidate () {
+      if (this.time.length > 0) {
```

Comment:
Need to change this test to 'if (this.time)' ... what is here now causes an error when using this dialog on existing assignments